src/MessageManager.cpp

```diff
@@ -208,7 +208,7 @@ void MessageManager::Unsubscribe( IMessageSubscriber* pSubscriber, MessageID m )
 void MessageManager::Broadcast( Message &msg ) const
 {
 	// GAMESTATE is created before MESSAGEMAN, and has several BroadcastOnChangePtr members, so they all broadcast when they're initialized.
-	if(this != NULL && m_Logging)
+	if(m_Logging) // 'this' won't be NULL
```

Comment:
I suppose I'll revert this to what it originally was. It's a little weird to me to allow calling this method from a NULL object.